src/main/java/ca/corefacility/bioinformatics/irida/processing/impl/FastqcFileProcessor.java

```diff
@@ -95,44 +98,62 @@ private void processSingleFile(SequenceFile sequenceFile) throws FileProcessorEx
 				.executionManagerAnalysisId(EXECUTION_MANAGER_ANALYSIS_ID)
 				.description(messageSource.getMessage("fastqc.file.processor.analysis.description", new Object[] {FastQCApplication.VERSION},
 						LocaleContextHolder.getLocale()));
+
+		IridaTemporaryFile iridaTemporaryFile = iridaFileStorageUtility.getTemporaryFile(fileToProcess);
```

Comment:
This processor gets a bit complicated.  Can you add comments through this file please about what's going on.  So up here something saying "we're getting a local copy of the files", down below "we're creating a temp directory", "we're running the fastqc modules", etc.

Reply:
Added in 117eedc7f06e586f08eed0fca9d520de62ca3c6f